src/System.Data.SqlClient/src/System/Data/SqlClient/SqlFileStream.Windows.cs

```diff
@@ -0,0 +1,685 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using System.Security.Permissions;
+using Microsoft.Win32.SafeHandles;
+using System.Buffers;
+
+namespace System.Data.SqlTypes
+{
+    public sealed partial class SqlFileStream : System.IO.Stream
+    {
+        // NOTE: if we ever unseal this class, be sure to specify the Name, SafeFileHandle, and 
+        // TransactionContext accessors as virtual methods. Doing so now on a sealed class
+        // generates a compiler error (CS0549)
+
+        // from System.IO.FileStream implementation
+        // DefaultBufferSize = 4096;
+        // SQLBUVSTS# 193123 - disable lazy flushing of written data in order to prevent
+        // potential exceptions during Close/Finalization. Since System.IO.FileStream will
+        // not allow for a zero byte buffer, we'll create a one byte buffer which, in normal
+        // usage, will not be used and the user buffer will automatically flush directly to 
+        // the disk cache. In pathological scenarios where the client is writing a single 
+        // byte at a time, we'll explicitly call flush ourselves.
+        internal const int DefaultBufferSize = 1;
+
+        private const ushort IoControlCodeFunctionCode = 2392;
+        private const int ERROR_MR_MID_NOT_FOUND = 317;
+        #region Definitions from devioctl.h
+        private const ushort FILE_DEVICE_FILE_SYSTEM = 0x0009;
+        #endregion
+
+        private System.IO.FileStream _m_fs;
+        private string _m_path;
+        private byte[] _m_txn;
+        private bool _m_disposed;
+        private static byte[] s_eaNameString = new byte[]
+        {
+            (byte)'F', (byte)'i', (byte)'l', (byte)'e', (byte)'s', (byte)'t', (byte)'r', (byte)'e', (byte)'a', (byte)'m', (byte)'_',
+            (byte)'T', (byte)'r', (byte)'a', (byte)'n', (byte)'s', (byte)'a', (byte)'c', (byte)'t', (byte)'i', (byte)'o', (byte)'n', (byte)'_',
+            (byte)'T', (byte)'a', (byte)'g', (byte) '\0'
+        };
+
+        public SqlFileStream
+            (
```

Comment:
I don't see this sort of indenting for parameter parenthesis in the other files in this assembly. I know the original source files were like this, but it would be worth following the CoreFX styles and keep the parens next to the arguments. There aren't many arguments here so they can go on one line. Also, unless there is a conflict, you can remove System.IO from the type names.

Reply:
Updated to normal styling as requested.